### DISCUSSION TITLE
Sharing images with some apps doesn't work.

### DISCUSSION THREAD

**Discussion by BjorniFPV** [deckerst/aves#542] [Q&A]
If i share an image out of aves via Whatsapp or Google Drive it works. 
But when I want to share an image via MEGA or KDE Connect it doesnt.
In other apps it is working. 

Maybe a permissions issue?

Running LineageOS 19.1

**Comment by deckerst** [comment]
In the app, there are instructions in the `About` page > `Bug Report` section. After following them, paste here your system information and attach your logs.

And when you write `sharing`, it's not clear to me if you mean from Aves to another app, or from another app to Aves.

If it's from Aves to another app, it depends on what the other app declares handling. Sometimes they handle the `share` intent ("share" in Aves), sometimes the `open` intent ("export" > "open with" in Aves), sometimes both.

Also, I'd need more details about what you mean by `work` / `doesn't work`.
